modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarCollapse.es.js

```diff
@@ -135,7 +135,7 @@ const CriteriaSidebarCollapse = ({
 						</a>
 
 						{active && (
-							<div>
+							<div className="sidebar-collapse-body">
 								<p className="pt-3 px-4 text-secondary">
```

Comment:
Hi Clara! Nice catch this bug 😃 

I wonder if it makes sense to leave the paragraph with the hint at the top, outside the scroll behaviour. I don't know how was originally developed... just a thought
What do you think?

![image](https://user-images.githubusercontent.com/19485114/125470020-2c7bdf41-40ae-48a6-b6fd-27f678167649.png)

Reply:
Hey @beltranrengifo, I put my comment below without reading yours, sorry 😛. What do you think of what I pointed? If we do that, the hint wouldn't be hidden.

Reply:
@beltranrengifo before the commit that broke the scroll we didn't have the paragraph at the top of the collapsable body  so I have no reference here. I did it like this because it made more sense to me to make all of the collapsable content scrollable but I see your point. We can discuss it with the frontend team a see what majority prefers.